caas-investor.html

```diff
@@ -28,6 +28,18 @@
     ></iron-ajax>
 
     <iron-ajax
+      id="getInvestorContracts"
+      url="[[apiEndpoint]]/investors/[[investorId]]/contracts"
+      handle-as="json"
+      headers="[[_requestHeaders]]"
+      method="GET"
+      last-response="{{_investorContractsData}}"
+      on-response="_handleGetInvestorContractsResponse"
+      on-error="_handleGetInvestorContractsError"
+      auto="[[accessToken]]"
```

Comment:
auto should probably be replaced?

`autoGetContract` property that can be set from parent. 
Another property that is computed(autoGetContract, accesstoken), if both true, set auto true?